src/components/ShareSaveQuery/shortenApi.js

```diff
@@ -6,11 +6,13 @@ export default ({ stats, queryName, sqon, loggedInUser }) => {
   let { Files, Participants, Families, Size } = stats;
   let alias = queryName || sqonToName({ filters: sqon });
 
+  const jwt = localStorage.getItem('EGO_JWT');
```

Comment:
`jwt` is available in global state `loggedInUserToken`, we should probably grab it from there to maintain the source of truth.